cmd/tempo/app/app.go

```diff
@@ -125,6 +129,8 @@ func (t *App) setupAuthMiddleware() {
 		noGRPCAuthOn := []string{
 			"/frontend.Frontend/Process",
 			"/frontend.Frontend/NotifyClientShutdown",
+			"/tempopb.BackendScheduler/Next",
```

Comment:
This was insufficient to prevent this endpoint from requiring auth.  Currently, the workerID is passed as the tenant to satisfy the call requirements, but it is not used on the server.

Reply:
I tried again and it worked.  :shrug: